### ISSUE TITLE
Ability to create type interface routes

### ISSUE THREAD

**Issue by f5-applebaum** [F5Networks/f5-declarative-onboarding#225]
For GCP, need to be able to configure type interface routes.

ex. 
tmsh create sys management-route mgmt_gw network ${MGMTGATEWAY}/32 type interface"
tmsh create net route ext_gw_interface network ${INT1GATEWAY}/32 interface external

From:
https://github.com/F5Networks/f5-google-gdm-templates/blob/v3.11.1/supported/failover/same-net/via-lb/3nic/existing-stack/payg/f5-existing-stack-same-net-cluster-payg-3nic-bigip.py#L524

https://github.com/F5Networks/f5-google-gdm-templates/blob/v3.11.1/supported/failover/same-net/via-lb/3nic/existing-stack/payg/f5-existing-stack-same-net-cluster-payg-3nic-bigip.py#L529

**Comment by dstokesf5** [comment]
Thank you for your feedback. This has been added to our internal product backlog as AUTOTOOL-2471.

**Comment by seattlevine** [comment]
The equivalent of
tmsh create net route ext_gw_interface network ${INT1GATEWAY}/32 interface external
is already supported by the 'target' property on the Route class. This was added in DO 1.14 to resolve #123

**Comment by seattlevine** [comment]
The equivalent of
```
tmsh create sys management-route mgmt_gw network ${MGMTGATEWAY}/32 type interface"
```
is already supported by the 'type' property of the ManagementRoute class.

**Comment by seattlevine** [comment]
However, there is a bug in the handling of ManagementRoute. We'll address that.

**Comment by dstokesf5** [comment]
This issue has been resolved with the 1.23.0 release of Declarative Onboarding.
